client/my-sites/plugins/index.node.js

```diff
@@ -0,0 +1,22 @@
+import { makeLayout, ssrSetupLocale } from 'calypso/controller';
+import {
+	browsePlugins,
+	browsePluginsOrPlugin,
+	fetchPlugins,
+	fetchPlugin,
+	fetchCategoryPlugins,
+} from './controller-2';
+
+export default function ( router ) {
+	router(
```

Comment:
Do we also need a route for the search endpoint - so that visitors can search for plugins?

Reply:
We link to popular searches so this is likely beneficial.

BTW We need to check these links are seo friendly. I think we use clickable spans with link roles or something.

Reply:
>  I think we use clickable spans with link roles or something.
I've noticed this in the Search Box but also the Category links are spans with click handlers. We'll need to make them proper links and pass them through a localizing function.